test/integration/models/image/test_image.py

```diff
@@ -42,10 +42,24 @@ def test_image_create_upload(test_linode_client):
     label = get_test_label() + "_image"
     image = test_linode_client.image_upload(
         label,
-        "us-ord",
+        "us-east",
```

Comment:
`us-ord` is not available for testing image creation here in alpha. We can also revert back before merging into dev.

Reply:
we can probably use `get_region` helper function in conftest when it is ready.
e.g. `region = get_region(client, {"Functionality"})`

Reply:
Good idea! Added to TODO.